### PR TITLE
server: Check Schema - download schema on build #265

### LINKED ISSUE #265: Check is Swagger spec is compliant before running rules against it.
Some rules can give you false positives when swagger spec is itself invalid. Error messages in this case can be very confusing. 
I propose to validate spec and if it's not valid emit special violation (without trying to check it with other rules). Much like we do if spec cannot be parsed.

### PR DESCRIPTION
**Pull request by akryvko**
Closes #265

### CODE REVIEW

**Comment by maxim-tschumak** [comment]
Is it possible to make the download optional?
I think it would be better to still have a default `swagger-schema` and `json-schema` definitions locally and update them if developer wants to (doesn't happen really often, right?)

I'm a long distance commuter and work sometimes in the train (no internet connection). I have all the needed dependencies in the local cache but the download of the schemas will fail.

File: server/src/main/resources/schemas/json-schema.json [hunk 1]
```diff
@@ -1,150 +0,0 @@
-{
```
Comment (netme) [inline server/src/main/resources/schemas/json-schema.json]: Could you please also add these files to `.gitignore` then?

**Comment by akryvko** [comment]
Then let's keep these files in git. It will be more safe for offline usage and more predictable. I've added only a task for updating them.

**Comment by netme** [comment]
@akryvko sounds good. Could you please also update `README.md` and let's merge it!

**Comment by netme** [comment]
👍

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by netme** [comment]
👍

**Pull request merged**